tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,6 +241,66 @@ install_kata_tools_placeholder() {
 		"after CI picks up the gha yaml changes required to test that installation."
 }
 
+function update_worker_label(){
+	local workers
+	local label="$1"
+	workers="$(kubectl get nodes --no-headers | grep '\<worker\>' | awk '{ print $1 }')"
+	for nodename in $workers; do
+		if ! kubectl get "node/${nodename}" -o jsonpath='{.metadata.labels}' | \
+			grep "$label" >/dev/null; then
+			kubectl label node "$nodename" --overwrite "${label}"
+		fi
+	done
+}
+
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+
+	case ${SNAPSHOTTER} in
+		nydus) deploy_nydus_snapshotter ;;
+		*) >&2 echo "${SNAPSHOTTER} flavour is not supported"; exit 2 ;;
+	esac
+
+	echo "::endgroup::"
+}
+
+function deploy_nydus_snapshotter() {
```

Comment:
I suggest to use `yq` instead of `sed` to edit the yaml files on this function. You can the `ensure_yq` auxiliary function that will install yq in the environment if not already present.

Reply:
Fixed.